addons/web/static/src/views/fields/properties/properties_field.js

```diff
@@ -548,9 +569,22 @@ export const propertiesField = {
         return {
             context: dynamicInfo.context,
             columns: parseInt(attrs.columns || "1"),
+            hideAddButton: archParseBoolean(attrs.hideAddButton),
             hideKanbanOption: archParseBoolean(attrs.hideKanbanOption),
         };
     },
 };
 
 registry.category("fields").add("properties", propertiesField);
+
+async function actionAddProperty(env) {
```

Comment:
Maybe a small explanation?

"This action is meant to be called from a form client action and will let the user create properties, typically used along with the "hideAddButton" option of the properties field".

(Or something similar).